### ISSUE TITLE
TypeError in calltip widget

### ISSUE THREAD

**Issue by Reinis** [spyder-ide/spyder#17120]
### What steps will reproduce the problem?

Start Kite tutorial Step 1a by placing cursor on the marked line after `m` and type `a`. After autocompletion popup shows, type `t`.

### Traceback
```python-traceback
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/spyder/plugins/editor/widgets/codeeditor.py", line 3026, in show_hint_for_completion
    self.show_hint(
  File "/usr/lib/python3.10/site-packages/spyder/widgets/mixins.py", line 641, in show_hint
    self.show_tooltip(signature=html_signature, text=extra_text,
  File "/usr/lib/python3.10/site-packages/spyder/widgets/mixins.py", line 620, in show_tooltip
    self.tooltip_widget.show_tip(point, tiptext, cursor=cursor,
  File "/usr/lib/python3.10/site-packages/spyder/widgets/calltip.py", line 192, in show_tip
    point.setY(adjusted_point.y() - tip_height - padding)
TypeError: setY(self, int): argument 1 has unexpected type 'float'
```

## Versions

* Spyder version: 5.2.0 
* Python version: 3.10.1
* Qt version: 5.15.2
* PyQt5 version: 5.15.6
* Operating System: Linux 5.15.11-arch2-1

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.30.1 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.8.9 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.1.0 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
parso >=0.7.0;<0.9.0          :  0.8.2 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.10.0 (OK)
pylint >=2.5.0                :  2.12.2 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.3.1;<1.4.0          :  1.3.2 (OK)
pylsp_black >=1.0.0           :  None (OK)
qdarkstyle >=3.0.2            :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.1 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.2.1;<5.3.0      :  5.2.1 (OK)
qtpy >=1.5.0                  :  1.11.2 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  57.4.0 (OK)
sphinx >=0.6.6                :  4.3.2 (OK)
spyder_kernels >=2.2.0;<2.3.0 :  2.2.0 (OK)
textdistance >=4.2.0          :  4.2.2 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  0.10.7 (OK)
xdg >=0.26                    :  0.27 (OK)
zmq >=17                      :  22.2.1 (OK)

# Optional:
cython >=0.21                 :  0.29.26 (OK)
matplotlib >=2.0.0            :  3.5.1 (OK)
numpy >=1.7                   :  1.21.5 (OK)
pandas >=1.1.1                :  1.3.5 (OK)
scipy >=0.17.0                :  1.7.3 (OK)
sympy >=0.7.3                 :  1.9 (OK)
```

**Comment by ccordoba12** [comment]
Hey @Reinis, thanks for reporting. We'll fix this in our next version (5.2.2), to be released soon.

**Comment by nbelakovski** [comment]
I'm still seeing this in 5.2.2. Repro steps:

In the IPython console, type the following commands hitting return after each one:
1. import numpy as np
2. x = np.linspace(-5, 5)
3. x

Now type np.linspace and once you type `(` you'll get the error. Delete and retype the `(` and you'll get the error again

**Comment by ccordoba12** [comment]
@nbelakovski, yours is not the same error as this one, but it's pretty similar. I fixed it in PR #17256, which will part of our next version (5.3.0), to be released in a couple of months.

Please use Spyder with Python 3.9 in the meantime.
